README.rst

```diff
@@ -132,11 +134,16 @@ The above configuration is the minimal required setup. The full set of options s
       'ignored_exceptions': [],
       'transmit_global_variables': True,
       'transmit_local_variables': True,
+      'enforce_payload_size_limit': True, 
+      'log_payload_size_limit_breaches': True,
       'transmit_environment_variables:': True,
       'userversion': "Not defined",
       'user': None
   }
 
+'enforce_payload_size_limit' when enabled (default behaviour) will iteratively remove the largest global or local variable from the error message until the payload is below 128kb
```

Comment:
It might be worth mentioning that payloads over 128kb will not be processed by Raygun